### PR TITLE
Updated docs to reflect the changes to the bundle

### LINKED ISSUE #3: Update the Readme
The readme is outdated and needs to be updated

### PR DESCRIPTION
**Pull request by Nyholm**
This will close #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,60 +1,58 @@
 PHP-Cache cache-bundle [![Build Status](https://travis-ci.org/php-cache/cache-bundle.png?branch=master)](https://travis-ci.org/php-cache/cache-bundle)
 ====================
 
-#### Cache Bundle for Symfony 2
+#### Cache Bundle for Symfony 2.7 and above
 
-Symfony 2 library providing PSR-6 compliant cache services for the user. 
-It also lets you cache your sessions and routes.
+Symfony bundle helping you interrate your  PSR-6 compliant cache service with the framework.
```
Comment (cryptiklemur) [inline README.md]: interrate -> integrate

File: README.md [hunk 2]
```diff
@@ -1,60 +1,58 @@
 PHP-Cache cache-bundle [![Build Status](https://travis-ci.org/php-cache/cache-bundle.png?branch=master)](https://travis-ci.org/php-cache/cache-bundle)
 ====================
 
-#### Cache Bundle for Symfony 2
+#### Cache Bundle for Symfony 2.7 and above
 
-Symfony 2 library providing PSR-6 compliant cache services for the user. 
-It also lets you cache your sessions and routes.
+Symfony bundle helping you interrate your  PSR-6 compliant cache service with the framework. 
+It lets you cache your sessions and routes. It also provides an integration to the debug toolbar.
```
Comment (cryptiklemur) [inline README.md]: to the -> with the

**Comment by cryptiklemur** [comment]
Why remove the documentation on the doctrine integration?

**Comment by Nyholm** [comment]
Sorry, I thought the doctrine was removed.. I was wrong. Thank you!

File: README.md [hunk 3]
```diff
@@ -1,40 +1,60 @@
 PHP-Cache cache-bundle [![Build Status](https://travis-ci.org/php-cache/cache-bundle.png?branch=master)](https://travis-ci.org/php-cache/cache-bundle)
 ====================
 
-#### Cache Bundle for Symfony 2
+#### Cache Bundle for Symfony 2.7 and above
```
Comment (cryptiklemur) [inline README.md]: 2.6

File: README.md [hunk 4]
```diff
@@ -1,40 +1,60 @@
...
 
-The respective cache extensions will be required for your project.
-
-Redis uses the php redis extension.
 
 #### Requirements
 
 - PHP >= 5.6, < 7.1
-- Symfony >= 2.7, < 4.0 
+- Symfony >= 2.7, ~3.0
```
Comment (cryptiklemur) [inline README.md]: 2.6, ^3.0

File: README.md [hunk 5]
```diff
@@ -1,40 +1,60 @@
...
 
 - PHP >= 5.6, < 7.1
-- Symfony >= 2.7, < 4.0 
+- Symfony >= 2.7, ~3.0 
 - [Composer](http://getcomposer.org)
 
 #### To Install
 
+You need to install and enable this bundle and also a PSR-6 cache implementation. In the example below we use the
+[DoctrineAdapterBundle].
```
Comment (cryptiklemur) [inline README.md:20]: does this need empty brackets after?

File: README.md [hunk 6]
```diff
@@ -76,39 +95,20 @@ This bundle also provides router caching, to help speed that section up. To enab
...
 
-If you change any of your routes, you will need to clear all of the route_* keys in your cache.
-
-
-#### To Use
+If you change any of your routes, you will need to clear the cache. If you use a cache implementation that supports 
+tagging (implements [TaggablePoolTrait](https://github.com/php-cache/taggable-cache/blob/master/src/TaggablePoolInterface.php))
+you can clear the cache tagged with `routing`.
 
-To use this with doctrine's entity manager, just make sure you have `useResultCache` and/or `useQueryCache` set to true. If you want to use the user cache, just grab the service out of the container like so:
```
Comment (cryptiklemur) [inline README.md:89]: should put this back in

File: README.md (hunk 5 above)
Comment (Nyholm) [inline README.md:20]: No, not if you write [DoctrineAdapterBundle]: url at the bottom

File: README.md (hunk 6 above)
Comment (Nyholm) [inline README.md:89]: Is this needed? I don't see the use case for using them directly. Can you think of any?

Comment (cryptiklemur) [inline README.md:89]: If you don't specify that the user has to use these when building a query, it won't cache queries or results

Comment (Nyholm) [inline README.md:89]: Yes, of course. Sorry, I misunderstood you. 

I've added this comment and an example.

**Comment by cryptiklemur** [comment]
Apparently, theres merge conflicts

**Comment by Nyholm** [comment]
I dont think it is a merge conflict. I configured the repos to "Protect master". Which means we cant merge until travis passes. (I thought that was generally a good thing...) Remove it if you don't like it. 

Have a look at #9, that one will fix travis. After that one is merge I can rebase this branch.

**Comment by Nyholm** [comment]
Thank you for squashing the commits. 
I've tried to update docs on all repos. There are some left on the doctrine adapter bundle though.

**Comment by cryptiklemur** [comment]
:+1:

**Pull request merged**